integration/app/test-demo/src/app.tsx

```diff
@@ -0,0 +1,24 @@
+import { lightTheme } from 'amazon-chime-sdk-component-library-react';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
Can you move these two imports from `react` and `react-router-dom` to the top?

Reply:
Updated.